src/styles/settings/_font_map.scss

```diff
@@ -2,7 +2,8 @@
 $font-sizes: (
   "xs": 1rem, // 16px
   "sm": 1.25rem, // 20px
-  "md": 1.875rem, // 30px
+  "mobile-heading": 1.5rem, // 24px
```

Comment:
Ah, the age-old trouble with T-shirt sizes, what to do with in-between values... If only there were such a thing as extra medium 🤔 

I'm good with `mobile-heading`. Better to add one weird size than to have to reshuffle all the others to fit the scale.